setup.cfg

```diff
@@ -18,3 +18,9 @@ universal = 1
 
 [upload_sphinx]
 upload-dir = docs/_build/html
+
+[pycodestyle]
+ignore = E501, W504, E225, E305
+
+[pydocstyle]
+ignore = E305, D400, D401, D104, D415
```

Comment:
Currently, ignoring `pycodestyle` and `pysocstyle` errors. Opened another issue -> https://github.com/inveniosoftware/dojson/issues/205.